src/middleware/validation.ts

```diff
@@ -0,0 +1,71 @@
+import { body } from "express-validator";
+import Recruiter from "../models/recruiter";
+import candidate from "../models/candidate";
+
+export const validateSignup = 
+    [
+		body('email', 'Please enter a valid email.')
+			.isEmail()
+			.custom(async (value) => {
+				const user = await Recruiter.findOne({ email: value }).exec();
```

Comment:
check if exec() is really required, we are using await and also waiting for a thenable promise, so check it once.